### ISSUE TITLE
compass create using blueprint raises an error

### ISSUE THREAD

**Issue by skwrl** [Compass/compass#504]
On debian with a fresh compass install, creating a new project using blueprint raises a fatal error: NoMethodError

Here's the command I used

```
compass create my_project --using blueprint --trace
```

And here's the result: https://gist.github.com/1124821 

```
compass version                                                                                                 
Compass 0.11.5 (Antares)
Copyright (c) 2008-2011 Chris Eppstein
```

**Comment by yevgenko** [comment]
I'm encountering exactly same issue(output), but with [Susy](http://susy.oddbird.net/)...

**Comment by skwrl** [comment]
Is blueprint working for you ?
Cause I tried susy and oddly enough it works fine here.

```
gem install compass-susy-plugin
Successfully installed compass-susy-plugin-0.9
```

Then I used this command to create the project:

```
compass create my_project -r susy -u susy
```

**Comment by yevgenko** [comment]
Oh, default susy projects works fine for me, but I'm actually looking
at [320+susy](https://github.com/ericam/320-susy) which throwing the
same error you reporting...

**Comment by yevgenko** [comment]
and same error with blueprint...

**Comment by yevgenko** [comment]
I was able to get [320+susy](https://github.com/ericam/320-susy) working by commenting out `@import compass/layout` in sass/_base.sass and clearing .sass-cache

**Comment by awmartin** [comment]
I've encountered the same error with an already existing compass+blueprint project. I attempted `compass compile` with version 0.11.5 and the same NoMethodError appeared.

**Comment by chriseppstein** [comment]
this is a sass bug. Please downgrade to sass 3.1.5 as a work around.

```
gem uninstall sass --version 3.1.6
gem install sass --version 3.1.5
```

**Comment by chriseppstein** [comment]
I opened a bug against sass: https://github.com/nex3/sass/issues/149 Closing this here.

**Comment by chriseppstein** [comment]
fixed in sass 3.1.7
